src/reactor/fns.js

```diff
@@ -330,22 +331,21 @@ export function evaluate(reactorState, keyPathOrGetter) {
   }
 
   // Must be a Getter
-  // if the value is cached for this dispatch cycle, return the cached value
-  if (isCached(reactorState, keyPathOrGetter)) {
-    // Cache hit
-    return evaluateResult(
-      getCachedValue(reactorState, keyPathOrGetter),
-      reactorState
-    )
-  }
 
-  // evaluate dependencies
-  const args = getDeps(keyPathOrGetter).map(dep => evaluate(reactorState, dep).result)
-  const evaluatedValue = getComputeFn(keyPathOrGetter).apply(null, args)
+  const cache = reactorState.get('cache')
+  var cacheEntry = cache.lookup(keyPathOrGetter)
```

Comment:
If the cache is a hit on a `getter` then subsequent `evalutes` will always think the value is a hit.  

Maybe I am missing something here, but it seems like this is wrong in a lot of cases.

```javascript
// store state 1
var result1 = reactor.evaluate(getter)
reactor.dispatch('action')
// store state 2
var results2 = reactor.evaluate(getter)
// it would seem that since the reactor cache has an entry for `getter` than it will always be a hit
```

Is there a test case to verify this?